### DISCUSSION TITLE
Feature suggestion: highlight rooms accessible from each room (sanctum)

### DISCUSSION THREAD

**Discussion by slaren** [Lailloken/Exile-UI#178] [Ideas & Feedback]
When everything else is equal, one common strategy when choosing what room to move to is to select the room that leaves more rooms accessible in the future, since this maximizes your options. It could be useful to have a tool to do this automatically, so that when you hover over a specific room it shows you what rooms are accessible from that point. For example, it could show something like this (when hovering over the bottom room of the second column):
![image](https://user-images.githubusercontent.com/2141330/210086057-0ac0b460-7378-4ac2-b56e-5dd4bd00a725.png)

**Comment by Lailloken** [comment]
Sounds really interesting but also quite tricky to accomplish by screen-reading. I'll think about it a bit to see if there's a reliable way to implement this.

**Comment by Lailloken** [comment]
Don't know if you're still interested (or generally invested in PoE), but someone brought up this feature-concept again recently. I was more confident this time around and played around with it. The first public release of the feature is available in [v1.55.0](https://github.com/Lailloken/Lailloken-UI/releases/tag/v1.55.0).
